api/allennlp_demo/adversarial_binary_gender_bias_mitigated_roberta_snli/requirements.txt

```
@@ -0,0 +1,4 @@
+git+git://github.com/allenai/allennlp.git
+git+git://github.com/allenai/allennlp-models.git
```

Could do this for now to make sure you get the latest commit:

```suggestion
git+git://github.com/allenai/allennlp-models.git@bdf82a18777f2ac37439621c712b2114db14c3b1
```